devices.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": "Fairphone 2",
+    "adb": {
+      "ro.product.device": [ "FP2" ],
+      "ro.build.id": ["LVY48I"]
```

Comment:
Those values should be the ones from stock Android ROM you will use when pulling the blobs. I have a feeling it's not the case :)